docs/_docs/04_reference/config-blocks-and-attributes.md

```diff
@@ -1285,9 +1285,8 @@ root level `terragrunt.hcl` file, you can set `skip = true`:
 skip = true
 ```
 
-The `skip` flag must be set explicitly in terragrunt modules that should be skipped. If you set `skip = true` in a
-`terragrunt.hcl` file that is included by another `terragrunt.hcl` file, only the `terragrunt.hcl` file that explicitly
-set `skip = true` will be skipped.
+The `skip` flag can be inherited from an included `terragrunt.hcl` file if `skip` is defined there, unless it is
```

Comment:
I think may be helpful to add an integration test for this case, when the skip flag is inherited from include